CIP-0068/README.md

```diff
@@ -331,25 +330,53 @@ We want to bring the metadata of the RFT `d5e6bf0500378d4f0da4e8dde6becec7621cd8
+review prior to official acceptance. Newly proposed `asset_name_labels` SHOULD NOT be added to 
+[CIP-0067](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0067) until the accompanying CIP has matured 
+through the community review and feedback stage to a point that it is considered in the `Under Review` status and is
+assigned a tentative CIP number by the CIP Editors panel.
+
+A brief reference to new asset classes MAY be added to this document after the accompanying CIP achieves the `accepted`
+status. Documentation describing these token asset classes MUST be fully encapsulated within their individual CIPs and a
+link MUST be provided to that CIP within this document.
+
+If a modification or change is deemed necessary to one of the asset classes contained within this document, namely Asset
+Name Labels: 100, 222, 333, or 444; which do not fundamentally change the nature, use, or reference of the tokens MAY
+be made as a modification of this document. However, any change proposed that presents a non-backwards compatible change
+MUST include an accompanying `version` field iteration and both specifications for the proposed, current, and historical
+versions of the format MUST be maintained in a logical way to assist future implementors who may encounter a version of
+these tokens from any point in time.
+
+If a change is proposed that would fundamentally alter the nature of one or more of the `asset_name_labels` and their 
+associated tokens contained within this document, namely Asset Name Labels: 100, 222, 333, or 444; these changes
+MUST be submitted via a new, separate CIP with its own justification, implementation, rationale, and community review
+prior to official acceptance. These separate CIPs MUST include a plan for the obsolescence of any previous versions of
+the affected tokens. `asset_name_labels` MUST only be marked obsolete once a modifying CIP achieves the `accepted` status.
 
 ## Path to Active
 
-- Agree on a binary encoding for asset name labels in [CIP-0067](https://github.com/cardano-foundation/CIPs/pull/298).
-- Get support for this CIP by wallets, explorers, tools, minting platforms and other 3rd parties.
-- Minimal reference implementation making use of [Lucid](https://github.com/spacebudz/lucid) (off-chain), [PlutusTx](https://github.com/input-output-hk/plutus) (on-chain):
-[Implementation](./ref_impl/)
-- Open-source more practical implementations/projects which make use of this CIP.
+- [X] Agree on a binary encoding for asset name labels in [CIP-0067](https://github.com/cardano-foundation/CIPs/pull/298).
+- [X] Get support for this CIP by wallets, explorers, tools, minting platforms and other 3rd parties.
+- [X] Minimal reference implementation making use of [Lucid](https://github.com/spacebudz/lucid) (off-chain), [PlutusTx](https://github.com/input-output-hk/plutus) (on-chain): [Implementation](./ref_impl/)
+- [ ] Open-source more practical implementations/projects which make use of this CIP.
```

Comment:
Does Nebula count for this step? 
https://github.com/spacebudz/nebula/tree/main

Reply:
I'll count it, I think we can well and truly consider CIP-68 to be fully "accepted" at this point.